test/WebSites/ContentNegotiationWebSite/Controllers/FallbackOnTypeBasedMatchController.cs

```diff
@@ -40,7 +40,7 @@ public IActionResult FallbackGivesNoMatch(int input)
         public IActionResult OverrideTheFallback_UsingCustomFormatters(int input)
         {
             var objectResult = new ObjectResult(input);
```

Comment:
my bad there is already one. I would still say to add a functional test which tests the acutal customer scenario.